jf_agent/git/github.py

```diff
@@ -290,7 +307,7 @@ def _get_merge_commit(client: GithubClient, pr, strip_text_content, redact_names
         )
         if api_merge_commit:
             return _normalize_commit(
-                api_merge_commit, pr['base']['repo'], strip_text_content, redact_names_and_urls
+                api_merge_commit, pr['base']['repo'], pr['head']['ref'], strip_text_content, redact_names_and_urls
```

Comment:
Adding a branch name for the commit being created here, but I am not sure if we want the base branch or head branch's name here from the pr.